sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/SplittableDoFnTest.java

```diff
@@ -304,6 +305,7 @@ public void testOutputAfterCheckpointBounded() {
   @Test
   @Category({ValidatesRunner.class, UsesUnboundedSplittableParDo.class})
   public void testOutputAfterCheckpointUnbounded() {
+    p.getOptions().as(TestPipelineOptions.class).setTestTimeoutSeconds(900L);
```

Comment:
This value is runner-specific. For many configurations a much smaller number is appropriate. This number is probably safe for all runners so it OK. But can you set it in the gradle file to some large number that will work for all the ValidatesRunner tests? Probably 900 is safe for any of them since they are small compliance tests and none are high load.

Reply:
It turns out this value doesn't help the test to pass. The `testOutputAfterCheckpointUnbounded ` and `testBundleFinalizationOccursOnUnboundedSplittableDoFn ` are just hanging on dataflow streaming runner v2. I'll revert the change here.